plugins/main/public/components/endpoints-summary/table/agents-table.tsx

```diff
@@ -326,7 +346,7 @@ export const AgentsTable = compose(
                       label: 'dateAdd',
                       description: 'filter by registration date',
                     },
-                    { label: 'id', description: 'filter by id' },
+                    { label: 'id', description: 'filter by ID' },
```

Comment:
**nitpick**: review other references to `filter by id`. If we change to `ID` here, then we could need to change in other sides to be consistent.

Reply:
I updated in other places where id was `in` lowercase.